### DISCUSSION TITLE
Angular 17 support

### DISCUSSION THREAD

**Discussion by f2ostie** [primefaces/community#610] [PrimeNG Blocks] [answered]
Have to say I'm pretty dissapointed. Just bought PrimeNG Blocks. Running the project locally and I noticed that the version of angular is 15. That is 2 major versions behind the state of the art. And I did not see this fact on your website before purchase.

Is there a timetable of when PrimeNG Blocks will be updated? Maybe even use standalone components as this would be an ideal usecase.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Blocks will be updated to v17 soon. However, since the blocks are copy-paste HTML elements, the only change will be inside the package.json file by upgrading v17. Code will stay the same. You can still use them no matter what version you're using.

**Comment by digit81** [comment]
Hi, 
I also have a licence to PrimeBlocks and have questions about project roadmap
When do you plan upgrade to newest PrimeNG in PrimeBlocks?
Do you plan to add some new blocks?

Regards

**Reply by cetincakiroglu** [reply]
Hi,

Yes new blocks are in progress right now. We don't have an exact time but probably, new blocks will be released in Q3. You can use blocks in v17. It's independent of the Angular version. Blocks from v15 will work in v17 since it's just copy-paste HTML code.

**Comment by GitStorageOne** [comment]
Also was disappointed (at least that was my first impression).  

> the only change will be inside the package.json file
Updated primeblocks using Angular update guide: https://update.angular.io/?l=3&v=15.0-17.0  
Then, additional changes was required in `package.json, angular.json, polyfills.ts`.  

It seems to work now, but I haven't tested it extensively, some minor visual bugs are definitely there.
Primeblocks is expected to be in sync with major Primeng releases (otherwise there should be a clear statement).
